exporter/stackdriver/examples/stats/main.go

```diff
@@ -49,13 +53,6 @@ func main() {
 	}
 	view.RegisterExporter(exporter)
 
-	// Create measures. The program will record measures for the size of
-	// processed videos and the nubmer of videos marked as spam.
-	videoSize, err := stats.Int64("my.org/measure/video_size", "size of processed videos", "MBy")
```

Comment:
How about let's bring that back in here instead of creating it as a global variable?

Reply:
why? we encourage users to declare these as globals. major justification for this change was to make declaring as globals easier so thought it appropriate to demonstrate this here.

Reply:
I mean, then sure, let's use it in not only main but say in another helper, otherwise as it is, doesn't look justified to be a global.

Reply:
I will extract the record into a separate function. I think having multiple files for an example would be too much.

Reply:
In deed. Not multiple files though, just multiple functions.